### PR TITLE
Fixed $ -> %

### PR DESCRIPTION
**Pull request by tomas-pluskal**

### CODE REVIEW

File: article.tex [hunk 1]
```diff
@@ -115,7 +115,7 @@
 \author[
    addressref={ebi},
    email={steinbeck@ebi.ac.uk}
-]{\inits{CS}\fnm{Christoph}~\snm{Steinbeck}}
+]{\inits{CS}\fnm{Christoph} \snm{Steinbeck}}
```
Comment (egonw) [inline article.tex]: I had those tildes there intentionally...

File: article.tex [hunk 2]
```diff
@@ -879,7 +879,10 @@ \section*{Authors contributions}
 \section*{Acknowledgements}
 The authors acknowledge the great number of people who have contributed smaller
 and larger contributions to the CDK library. A full list of contributors is
-found in the AUTHORS file~\cite{AUTHORS}. OS acknowledges support from the Swedish strategic research programs eSSENCE and Swedish e-Science Research Center (SeRC). TP is a Simons Foundation Fellow of the Helen Hay Whitney Foundation.
+found in the AUTHORS file~\cite{AUTHORS}. OS acknowledges support from the
+Swedish strategic research programs eSSENCE and Swedish e-Science Research
+Center (SeRC). TP is a Simons Foundation Fellow of the Helen Hay Whitney
+Foundation.
```
Comment (egonw) [inline article.tex]: Nothing really changed here, or?

**Comment by egonw** [comment]
I applied the ORCID and $ parts of the patch and left comments for the rest.

File: article.tex (hunk 2 above)
Comment (tomas-pluskal) [inline article.tex]: Just changed formatting, no change to the contents

File: article.tex (hunk 1 above)
Comment (tomas-pluskal) [inline article.tex]: Ah, sorry. I thought it was by mistake.

Comment (egonw) [inline article.tex]: It's the LaTeX implementation of the non-breaking space :)

**Pull request closed without merging**